src/main/java/uk/gov/companieshouse/limitedpartnershipsapi/controller/PartnershipController.java

```diff
@@ -47,7 +47,7 @@ public ResponseEntity<Object> createPartnership(
         var transactionId = transaction.getId();
         var logMap = new HashMap<String, Object>();
         logMap.put(URL_PARAM_TRANSACTION_ID, transactionId);
-
+        
```

Comment:
Spaces only added here. You can just revert all the changes to this file.

Reply:
file reverted